lib/chai/core/assertions.js

```diff
@@ -235,55 +346,144 @@ module.exports = function (chai, _) {
   Assertion.addChainableMethod('a', an);
 
   /**
-   * ### .include(value)
+   * ### .include(val[, msg])
+   *
+   * When invoked as a method, and when the target is a string, `.include`
+   * asserts that the given string `val` is a substring within the target
+   * string.
+   *
+   *     expect('foobar').to.include('foo');
+   *
+   * When invoked as a method, and when the target is an array, `.include`
+   * asserts that the given `val` is a member of the target array. By default,
+   * strict (`===`) equality is used to compare members.
+   *
+   *     expect([1, 2, 3]).to.include(2);
+   *
+   * When invoked as a method, and when the target is an object, `.include`
```

Comment:
I think that maybe we could have a more general sentence telling people the main goal of this assertion which is to check if a value includes another one and then detail both `method` and `property` behaviors.